ansible/rancher/downstream/custom_cluster/cleanup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+. vars.env
+
+# destroy empty cluster
+tofu -chdir="$TERRAFORM_CLUSTER_SOURCE" destroy -auto-approve -var-file="$TFVARS_FILE" &
+
+# destroy nodes
+tofu -chdir="$TERRAFORM_NODE_SOURCE" destroy -auto-approve -var-file="$TFVARS_FILE" &
```

Comment:
I'm not sure this file is necessary within ansible. It seems to just be a script that is coupled to tofu -- but what if I provided VMs separately and didn't manage them via tofu? Perhaps we can just include this in the relevant Tofu module(s)' README(s).